lib/api/formtools/grid.js

```diff
@@ -0,0 +1,263 @@
+var linz = require('../../../');
+    async = require('async'),
+    clone = require('clone');
+
+(function  () {
```

Comment:
This is to allow internal access to the functions within the scope of this file using self executable function to encapsulate this scope and only expose the functions that are accessible outside of this scope.